src/gmt.c

```diff
@@ -340,25 +343,25 @@ int main (int argc, char *argv[]) {
-			fprintf (stderr, "  --show-cores        Show number of available cores.\n");
-			fprintf (stderr, "  --show-datadir      Show directory/ies with user data.\n");
-			fprintf (stderr, "  --show-dataserver   Show URL of the remote GMT data server.\n");
-			fprintf (stderr, "  --show-doi          Show the DOI for the current release.\n");
-			fprintf (stderr, "  --show-library      Show path of the shared GMT library.\n");
-			fprintf (stderr, "  --show-modules      Show all modern module names.\n");
-			fprintf (stderr, "  --show-modules-core Show all modern module names (core only).\n");
-			fprintf (stderr, "  --show-plugindir    Show directory for plug-ins.\n");
-			fprintf (stderr, "  --show-sharedir     Show directory for shared GMT resources.\n");
-			fprintf (stderr, "  --show-userdir      Show full path of user's ~/.gmt dir\n");
-			fprintf (stderr, "  --version           Print GMT version number.\n\n");
+			fprintf (stderr, "  --help                        List descriptions of available GMT modules.\n");
+			fprintf (stderr, "  --new-script[-classic][=L]    Write GMT modern mode or classic mode script template to stdout.\n");
+			fprintf (stderr, "                                Optionally specify bash|csh|batch [Default is current shell].\n");
+			fprintf (stderr, "  --new-glue=name               Write C code for external supplements to glue them to GMT.\n");
+			fprintf (stderr, "  --show-bindir                 Show directory with GMT executables.\n");
+			fprintf (stderr, "  --show-citation               Show the most recent citation for GMT.\n");
+			fprintf (stderr, "  --show-classic                Show all classic module names.\n");
+			fprintf (stderr, "  --show-classic-core           Show all classic module names (core only).\n");
+			fprintf (stderr, "  --show-cores                  Show number of available cores.\n");
+			fprintf (stderr, "  --show-datadir                Show directory/ies with user data.\n");
+			fprintf (stderr, "  --show-dataserver             Show URL of the remote GMT data server.\n");
+			fprintf (stderr, "  --show-doi                    Show the DOI for the current release.\n");
+			fprintf (stderr, "  --show-library                Show path of the shared GMT library.\n");
+			fprintf (stderr, "  --show-modules                Show all modern module names.\n");
+			fprintf (stderr, "  --show-modules-core           Show all modern module names (core only).\n");
+			fprintf (stderr, "  --show-plugindir              Show directory for plug-ins.\n");
+			fprintf (stderr, "  --show-sharedir               Show directory for shared GMT resources.\n");
+			fprintf (stderr, "  --show-userdir                Show full path of user's ~/.gmt dir\n");
+			fprintf (stderr, "  --version                     Print GMT version number.\n\n");
 			fprintf (stderr, "if <module-options> is \'=\' we call exit (0) if module exist and non-zero otherwise.\n\n");
 			status = GMT_NOERROR;
 		}
```

Comment:
Please, do not change the formatting. Those tabs above should not be there. Only spaces guarantee a good alignment in all editor settings (and possible terminals)

Reply:
I'm very certain I used spaces when I made the changes, and double checked now. Do they appear as tabs to you?

Reply:
Hi @anbj, while it may not be used a lot I am OK with accepting your contribution.  However, I'd like to see a few changes:

1. Split the new-script usage message into two separate lines, one for classic and the original one, as is done for other classic versions of options.
2. Use less spaces: Only 1 after the longest of all entries, which after splitting into two lines will remain the show-modules-core I think. As per #5341, we want to keep things tighter.
3. You need to add an entry in the gmt.rst file as well for documentation.